source/logic/bounties.js

```diff
@@ -113,7 +113,7 @@ async function addCompleters(bounty, guild, completerMembers, runMode) {
 	}
 
 	if (validatedCompleterIds.length < 1) {
-		throw `No new turn-ins were able to be recorded. Bots cannot turn-in bounties. ${bannedIds.length ? ' The completer(s) mentioned are currently banned.' : ''}`;
+		throw `No new turn-ins were able to be recorded. You cannot turn-in your own bounties or credit bots. ${bannedIds.length ? ' The completer(s) mentioned are currently banned.' : ''}`;
```

Comment:
It felt like I was stumbling a bit while reading this, but I think this clarification is worth keeping in some regard. Maybe "You cannot credit yourself or bots for your own bounties?"
```suggestion
		throw `No new turn-ins were able to be recorded. You cannot credit yourself or bots for your own bounties. ${bannedIds.length ? ' The completer(s) mentioned are currently banned.' : ''}`;
```